### ISSUE TITLE
New rule: new-module-imports

### ISSUE THREAD

**Issue by Turbo87** [ember-cli/eslint-plugin-ember#86]
This rule should warn about:

```js
import Ember from 'ember';
```

and e.g.

```js
import Component from 'ember-component';
```

instead the new module imports should be used, e.g.:

```js
import Component from '@ember/component';
```

more information is available and linked at https://github.com/ember-cli/ember-rfc176-data
